packages/block-library/src/block/edit.js

```diff
@@ -31,11 +31,14 @@ import {
 } from '@wordpress/block-editor';
 import { useEffect, useState } from '@wordpress/element';
 
+const fullAlignments = [ 'full', 'wide', 'left', 'right' ];
+
 export default function ReusableBlockEdit( {
 	attributes: { ref },
 	setAttributes,
+	__unstableParentLayout,
```

Comment:
Nit: could we rename this prop `parentLayout` for use in the function? That way we only need to change it in one place when it's stabilised 😄